### PR TITLE
Fix Subsurface Scattering

### LINKED ISSUE #42869: Vulkan: SSS Does not seem to work
**Godot version:**

Godot v4.0.dev.calinou.94e5f66cb

**OS/device including version:**

Vulkan, RTX2080 Driver: 456.71 Windows 10 64Bit

**Issue description:**

when enabling subsurface scattering the mesh becomes transparent.

SSS_Issue

**Steps to reproduce:**
Create a mesh, assign a material, enable subsurface scattering 

**Minimal reproduction project:**

[SSS.zip](https://github.com/godotengine/godot/files/5395558/SSS.zip)

### LINKED ISSUE #48600: Can't render cheese in Godot 4 (material tester demo project)
**Godot version:**
Commit: 6e3f47983cabb5f162eaec075d810f717985beda

**OS/device including version:**
Windows 10, Vulkan, Nvidia 3070

**Issue description:**
Making our material tester demo work on Godot 4 there are two materials that result in the entire scene not rendering properly:

image

This happens with the Cheese and Marble materials, it looks like that it is subsurface scattering that is causing this.

**Steps to reproduce:**
Turn on subsurface scaterring on a material

**Minimal reproduction project:**
https://github.com/godotengine/godot-demo-projects/pull/616

### PR DESCRIPTION
**Pull request by reduz**
* Works again
* Transmittance also works again
* Removed the curve parameter, exp() function is good enough.

image

*Bugsquad edit:* Fixes #42869, fixes #48600.

### CODE REVIEW

File: doc/classes/Tree.xml [hunk 1]
```diff
@@ -96,14 +96,6 @@
 				Returns the column index at [code]position[/code], or -1 if no item is there.
 			</description>
 		</method>
-		<method name="get_column_expand_ratio" qualifiers="const">
```
Comment (clayjohn) [inline doc/classes/Tree.xml:99]: Unrelated to this PR

**Review by clayjohn — APPROVED** [review]

**Comment by BastiaanOlij** [comment]
This looks good to me

**Review by BastiaanOlij — APPROVED** [review]
As far as I can test this is working great :)

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**